en/advanced_features/package_delivery.md

```diff
@@ -0,0 +1,49 @@
+# Package Delivery in Mission
```

Comment:
This should all move to the development section somewhere - it is mostly interesting if you want understand how the current things are integrated, and how you might extend it. 

When yuou think it is done, let's get Beat to review this part.

Reply:
Ah yes, I think it would be nice to have a development section for control allocation, ekf, etc :thinking: 

Definitely not in the advanced_features though :thinking: